concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+For each board:
+- Briefly run through the Backlog column to decide if there are any issues we want to promote to "Tracking" status.
+- Run through all items in "Tracking" and decide which ones are highest priority, and who will work on them. They will become the issue’s assignee.
+
+From there, automation will aid the process -- any newly assigned issues will:
+- be moved to the Active column in the backlog board
+- automatically added to the Assigned column of the Active Sprint board
+
+This process repeats for every board, after which our Active Sprint board will be populated with our workload for the next two weeks, and our sprint planning is complete.
+
+### Daily Standup
+Our daily standups will take place every day we don’t have a longer meeting (such as sprint planning/retro) for half an hour. This is meant to be a quick check-in to ensure newly-opened issues get quick responses and blockers get resolved.
+
+First, if there are any new people on the call, we'll welcome them and allow them to ask questions or have discussions as needed.
+
+Next, we’ll have a look at all untriaged issues -- any that do not have a label, and are not part of a project. This will allow the whole team to discuss them and ensure they get moved to the appropriate backlog board and labeled correctly. This shouldn’t take more than 15 minutes, though that’s a soft limit -- if a good conversation is happening, it shouldn’t be cut off.
+
+Finally, if there's time, we'll dedicate some time to a quick review of our Current Sprint board. This will serve as an opportunity for developers to elaborate on any blockers and ask for reviews or re-reviews as needed. We should strive to do this every day if possible, but if prior discussion makes that infeasible, we don't _have_ to do it (there's always tomorrow). Project discussion is more important.
+
+### Sprint Retrospective
+Our final meeting is the sprint retrospective. This should be a quick half-hour meeting that takes the place of Monday’s standup where we discuss:
+- What we accomplished in the last sprint (perhaps with a demo, if appropriate)
+- What didn’t get done, and why (blockers? More complex than anticipated? etc)
+
+If there is leftover work on our Active Sprint board, we should decide whether we should continue working on it or demote it to a backlog board. Then, we move onto the next sprint and do it all over again.
+
+For the first few sprints we do, it may be worth setting some time aside for process feedback: how well is our agile approach working? How can we improve it? etc.
+
+## Drawbacks
+
+- The team will need to attend a few more meetings. The daily standups, as currently designed, lose their focus on individual areas of the project per day; this was useful in case an external contributor was interested in one specific area of the project. However, we hope the added free-form discussion time will allow for relevant questions to be asked and talked about anyway.
```

Comment:
As a newcomer, I actually think the free-form discussion is more valuable than the area-focus approach. The responsiveness and availability of the core team and the good use of chat make it easy to engage about a specific area as well.

Additionally, newcomer and external contributers are not present at most meetings and are not the majority of participants, so I think it is reasonable to structure meetings to maximally enable the core team and make adjustments as needed for new members when they do show up.